Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverOperatorAssociation.yaml

```diff
@@ -0,0 +1,27 @@
+imports:
+  Merchant: Domain.Types.Merchant
+  MerchantOperatingCity: Domain.Types.MerchantOperatingCity
+  Person: Domain.Types.Person
+
+DriverOperatorAssociation:
+  tableName: driver_operator_association
+
+  fields:
+    id: Id DriverOperatorAssociation
+    operatorId: Text
+    isActive: Bool
+    driverId: Id Person
+    associatedOn: Maybe UTCTime
+    associatedTill: Maybe UTCTime
+    createdAt: UTCTime
+    updatedAt: UTCTime
```

Comment:
`createdAt` and `updatedAt` can be removed, because they are generated automatically